scripts/docs_update_release.py

```diff
@@ -0,0 +1,216 @@
+
+new_idf_version = f"{idf_version[0]}.{idf_version[1]}{f'.{idf_version[2]}' if idf_version[2] else ''}"
+
+
+def _resolve_installer_type() -> str:
+    """Resolve the type of the installer
+        and return the string of new entry for the index.html
+    """
+    new_entry_ide = f"""            <div class="download-button">
+                    <form method="get" action="https://dl.espressif.com/dl/idf-installer/espressif-ide-setup-{ide_version}-with-esp-idf-{new_idf_version}.exe">
+                        <button class="button-espressif-ide">
+                            <i class="fa fa-download" aria-hidden="true"></i>
+                            <div>Espressif-IDE {ide_version} with ESP-IDF v{new_idf_version} - Offline Installer</div>
+                            <div>Windows 10, 11</div>
+                            <div>Size: {installer_size}</div>
+                        </button>
+                    </form>
+                </div>"""
+
+    new_entry_offline = f"""            <div class="download-button">
+                    <form method="get" action="https://dl.espressif.com/dl/idf-installer/esp-idf-tools-setup-offline-{new_idf_version}.exe">
+                        <button class="button-offline">
+                            <i class="fa fa-download" aria-hidden="true"></i>
+                            <div>ESP-IDF v{new_idf_version} - Offline Installer</div>
+                            <div>Windows 10, 11</div>
+                            <div>Size: {installer_size}</div>
+                        </button>
+                    </form>
+                  </div>"""
+
+    new_entry_online = f"""            <div class="download-button">
+                    <form class="download-form" method="get" action="https://dl.espressif.com/dl/idf-installer/esp-idf-tools-setup-online-{online_installer_version}.exe">
+                        <button class="button-online">
+                            <i class="fa fa-download" aria-hidden="true"></i>
+                            <div>Universal Online Installer {online_installer_version}</div>
+                            <div>Windows 10, 11</div>
+                            <div>Size: {installer_size}</div>
+                        </button>
+                    </form>
+                </div>"""
+    
+    if installer_type == 'espressif-ide':
+        return new_entry_ide
+    elif installer_type == 'offline':
+        return new_entry_offline
+    elif installer_type == 'online':
+        return new_entry_online
+
+
+def update_index():
+    """Update the index.html file with the new release of the installer"""
+    try:
+        with open(path.abspath(INDEX_PATH), "r") as index_file:
+            index_lines = index_file.readlines()
+    except FileNotFoundError as e:
+        raise SystemExit(f"Error opening file {INDEX_PATH} - {e}")
+
+    # find every element with the class "download-button"
+    elements = []
+    for i in range(0, len(index_lines)):
```

Comment:
Please use `enumerate()` instead of looping through indexes and accessing each element based on that.